### ISSUE TITLE
Money laundering insider information. Is the butterproject involved?

### ISSUE THREAD

**Issue by YIFYtorrents** [PopcornTime-CE/desktop#67]
## /r/PopcornTime/ Subreddit & Its Malware Forks - Part II

Popcorntime.sh currently sits at number four in a google search for “popcorn time” and how it got there is a mystery. It looks a lot like the other Popcorn Time websites and on the surface it seems like their version of Popcorn Time could be just as good as the next one. The Symantec Antivirus Software detected a [Sape.Heur Virus](https://archive.is/9id2c) packaged with the windows installer. This virus threatens your computer privacy, but surprisingly it is [reported](http://www.symantec.com/security_response/writeup.jsp?docid=2014-071714-3425-99) as "very low". However, looking a bit in to what’s going on with that virus it’s [evident](https://archive.is/u7apk#selection-141.1-145.57) you should stay clear of it.

There are many versions out there and not all of them has your best interest in mind. Here is a diagram that has been circulating on Reddit and attempts to put the current mess into an easy to understand format. It might help you navigate in the PT jungle out there.

![overview-1](https://cloud.githubusercontent.com/assets/16165181/13820107/ceb1ae38-eb93-11e5-922c-fba937c3d413.png)
1. Sape.Heur causes poor PC performance
2. It may hijack your web browsers or download additional viruses
3. Sape.Heur virus threatens your computer privacy.

Source: https://archive.is/u7apk#selection-141.1-145.57
Source: https://www.virustotal.com/en/file/8865340cf24731a28a8f2916e049dbbaff8424315817b39e0ddffafdb8f38318/analysis/1457901544

The bottom line is that you wan to stay far away from popcorntime.sh! If you feel like helping out you can report it to Google [here](https://www.google.com/safebrowsing/report_badware/). The more people report it the faster it’s going to be taken off Google search results.

**Comment by konfou** [comment]
From a quick look at a meld of the three Github-hosted forks (as of the commits https://github.com/PopcornTimeCommunity/desktop/commit/fcd39e11694817fe875f9f0481eabcce33c10170, https://github.com/popcorn-official/popcorn-desktop/commit/4f15b03b21411c86446e606d5e096310b8511e2a and https://github.com/PTCE-Public/popcorn-desktop/commit/5fda3f81a015cf12c8a0a52b9c829c1a3facaf7b) source code there are no important differences to the back-end.
So those building from source shall probably be fine with all three of them.

**Comment by YIFYtorrents** [comment]
> there are no important differences to the back-end.

that's totally wrong. The google cloud plugin is just available for [popcorntime.ag](http://popcorntime.ag)
